koku/masu/api/expired_data.py

```diff
@@ -42,8 +43,19 @@ def expired_data(request):
         simulate = False
     LOG.info("Simulate Flag: %s", simulate)
 
+    params = request.query_params
+    acceptabools = ["true", "True", "false", "False"]
+    line_items_only = params.get("line_items_only", "False")
+    if line_items_only not in acceptabools:
```

Comment:
First off, ❤️ the variable name `acceptabools`. And since this is an internal-only API this is not remotely critical, but one thing you could do here is call `.lower()` (or upper) on the `line_items_only` variable and just make sure it's `true` or `false`. That way someone could do `tRuE` as the param and it would still pass.

Reply:
Glad you appreciate the puns 😂 The `.lower()` is a wonderful idea. Super simple and elegant but catches a lot of edge cases. 👍

Reply:
I like the variable name, but if it's only being used once, it doesn't need to be assigned to a variable at all.

```
    if line_items_only not in ["true", "false"]:
```

Though, if you want to get _really_ clever, you can use the `json` module to parse the string into a native Python type for you. (It'll throw a `JSONDecodeError` if it's not "true" or "false" or some other valid JSON value. So, you'll need a try/except block around it.)


```
IPython 7.11.1 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import json                                                                                                                                        
In [2]: foo = "true"                                                                                                                                       
In [3]: json.loads(foo)                                                                                                                                    
Out[3]: True
```

Reply:
Not gonna lie, I was just trying to keep the `acceptabools` variable 😂 , but I can totally make this change.

Reply:
So, I realized as I was making the change that I also use the acceptabools in the error message here: https://github.com/project-koku/koku/pull/1727/files#diff-b344b463fb32d3ad4607281872704ecdR51

So, I just kept it in, but I can also just write out in the string `['true', 'false']` if we do want to get rid of it.